layouts/default.vue

```diff
@@ -215,6 +234,23 @@ export default {
       this.$colorMode.preference =
         this.$colorMode.value === 'dark' ? 'light' : 'dark'
     },
+    async updateNotifCount() {
+      if (
+        this.$auth.user.id &&
+        Date.now() - this.notifications.lastUpdated > 300000
```

Comment:
Is the delay really needed? 
It would be useful in the case where the notification amount is stored in the vuex store and persisted, but it is not the case, and the `created` method is only called once per element, so that auto update isn't working.

Reply:
The delay is needed because `this.updateNotifCount()` is called on route changes too. Storing the notification count in the vuex store seemed overkill because the notification count is only needed in `default.vue`.

Reply:
The store is never overkill, as it allows us to know for sure that this is dynamic and getting insights in the vue debug tools.
But I can't seem to find the place where you call the updateNotifCount.

Reply:
Its called right here:
https://github.com/venashial/knossos/blob/3c44b43c1f8128758ad92365727132dde5505f86/layouts/default.vue#L195

Do you think it is still worth migrating to the vuex store?

Reply:
It is still worth, as it will make it clearer on for the next one working on it :)

Reply:
Should I create a new vuex module just for notifications? The auth one already stores `userFollows`.

Reply:
Yup, creating modules is not pretty costy on the JS side

Reply:
Should I move `userFollows` while I'm at it? I could call the store something general like `user`

Reply:
Yup! `user` is perfect, and it will be used when we have our custom auth system

Reply:
Turns out `userFollows` isn't used anywhere. Not sure why/how it got into the vuex store.

Reply:
Everything is migrated to a vuex store in d527982.